pkg/daemon/ceph/osd/daemon.go

```diff
@@ -37,6 +37,7 @@ import (
 const (
 	pvcDataTypeDevice     = "data"
 	pvcMetadataTypeDevice = "metadata"
+	pvcWaldataTypeDevice  = "waldata"
```

Comment:
I'd suggest where you have `waldata` in the name, we just shorten the name to `wal`.
```suggestion
	pvcWalTypeDevice  = "wal"
```

Reply:
done